docs/content/installation/using-gcp-marketplace-package.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Using GCP Marketplace Ingress Controller
+description: "This document will walk you through the steps needed to deploy the NGINX Ingress Controller through the GCP Marketplace."
+weight: 2400
+doctypes: [""]
+toc: true
+docs: "DOCS-000"
+---
+
+This document will walk you through the steps needed to deploy and configure the NGINX Ingress Controller through the GCP Marketplace.
+
+## Product overview.
+
+The F5 NGINX Ingress Controller an implementation of a Kubernetes Ingress Controller for NGINX and NGINX Plus.
```

Comment:
```suggestion
The F5 NGINX Ingress Controller is an implementation of a Kubernetes Ingress Controller for NGINX and NGINX Plus.
```
Decided to take another look at the PR, this stuck out immediately - can't believe I missed it on the first pass.